includes/ApplHandlingByStr.php

```diff
@@ -49,14 +49,14 @@ function showStatus( $stat_str )
 function showFullStatus( $stat_str )
 {
 	$STR = explode ( ';', $stat_str );
-	for ( $i =  1;  $i < count ( $STR ) ; $i++ )
+	for ( $i =  0;  $i < count ( $STR ) ; $i++ )
 	{
 		$str = explode ( ',', $STR[$i] );
-		$usr = $str[0];
+		
 		
 		if( $str[1] == 'P' )
 		{
-			echo "<td><b>PENDING</b> by $usr</td>";
+			echo "<td><b>PENDING</b> by ".$str[0]."</td>";
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->